frontend/packages/substrate-react/src/dotsama/hooks/useEagerConnect.ts

```diff
@@ -18,7 +18,11 @@ export const useEagerConnect = (chainId: ParachainId): boolean => {
 
   useEffect(() => {
     if (parachainApi !== undefined && activate !== undefined && !hasTriedEagerConnect && extensionStatus === "initializing") {
-      activate(false);
+      // activate();
+      const usedWallet = localStorage.getItem("wallet-id");
+      if (usedWallet && usedWallet === "talisman" || usedWallet === "polkadot-js") {
```

Comment:
Possible to use enum or const for the strings?

e.g. 

`usedWallet === WALLETS.Talisman`

Reply:
Sure